app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -81,6 +84,15 @@ function ReviewIssuesStep({
 
   const { onFailedSubmissionAttempt } = useContext(FailedCaptureAttemptsContext);
   const { inPersonURL } = useContext(InPersonContext);
+
+  function DocTypeErrorMessage({ error }) {
```

Comment:
I wouldn't expect to see a nested component like this, especially since only `error` is passed as a prop and `remainingAttempts` is used from the outer scope. Could we either inline this, or extract it to a new component file and pass all of the props it needs?